.pipelines/templates/.builder-release-template.yaml

```diff
@@ -259,4 +259,4 @@ steps:
       vstsFeedPublish: 'CloudNativeCompute/AKS-Linux-VHD-Artifacts'
       vstsFeedPackagePublish: 'publishing-info-${{ parameters.artifactName }}'
       packagePublishDescription: 'VHD publishing info ${{ parameters.artifactName }}'
-      versionOption: minor
```

Comment:
minor nit: no unnecessary changes

Reply:
It shows the `versionOption` added back. I think this change might just be a false positive

Reply:
I think the newline was removed. anyways, let's get this merged.